SwiftFlow/CoreTypes/Store.swift

```diff
@@ -44,9 +53,11 @@ public protocol Store {
      Dispatches an action. This is the simplest way to modify the stores state.
 
      Example of dispatching an action:
-     ```swift
+     
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)